assignments/callbacks.js

```diff
@@ -2,29 +2,117 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function firstItem(arr, cb) {
   // firstItem passes the first item of the given array to the callback function.
+  
+  const item =cb(arr);
+  console.log(item);
 }
 
+function printItem(items){
+return items[0];;
+}
+
+firstItem(items,printItem)
+
+
+
+
+
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  
+  const item = cb(arr);
```

Comment:
We want to pass the length of `arr` into the callback. So for this, we would do:
```
const item = cb(arr.length);
```